### ISSUE TITLE
AgeBin set robustness

### ISSUE THREAD

**Issue by dtcitron** [BershteynLab/EMODAnalyzeR#53]
For emodplot.by_age_gender, there's an error if there are empty Age Bins. Should alter this to make it more robust. Either fill the rows with zeroes, or discount the unused Age Bins somehow.

**Comment by kaftand** [comment]
Agreed - this probably should have a different function name too. It really just filters the ages, which doesn't need to happen in the function. I propose changing the function to emodplot.by_gender and removing the filter.
